src/main/java/org/spongepowered/common/event/SpongeCommonEventFactory.java

```diff
@@ -473,6 +474,25 @@ public static InteractItemEvent callInteractItemEventSecondary(EntityPlayer play
         return event;
     }
 
+    public static boolean callDestructEntityEvent(Entity entity, @Nullable Cause cause) {
+        if (((net.minecraft.entity.Entity) entity).isDead && (!(entity instanceof EntityLivingBase) || entity instanceof EntityArmorStand)) {
+            MessageChannel originalChannel = MessageChannel.TO_NONE;
+            if (cause == null) {
+                cause = Cause.of(NamedCause.source(entity.getWorld()));
+            }
+            DestructEntityEvent event = SpongeEventFactory.createDestructEntityEvent(
+                    cause, originalChannel, Optional.of(originalChannel), new MessageEvent.MessageFormatter(),
+                    entity, true
+            );
+            SpongeImpl.getGame().getEventManager().post(event);
+            if (!event.isMessageCancelled()) {
+                event.getChannel().ifPresent(channel -> channel.send(entity, event.getMessage()));
+            }
+        }
+
+        return true;
```

Comment:
It doesn't look like the return value has any purpose, it's unused in the code that calls this method